dbt_subprojects/dex/models/trades/base/platforms/stablebase_base_base_trades.sql

```diff
@@ -8,6 +8,11 @@
     unique_key = ['block_date', 'blockchain', 'project', 'version', 'tx_hash', 'evt_index']
 ) }}
 
+-- Id mapping:
+-- 0 USDbC https://basescan.org/token/0xd9aaec86b65d86f6a7b5b1b0c42ffa531710b6ca
+-- 1 DAI https://basescan.org/token/0x50c5725949a6f0c72e6c4a641f24049a917db0cb
+-- 2 axlUSDC https://basescan.org/token/0xeb466342c4d449bc9f53a865d5cb90586f405215
```

Comment:
Hi @PatelPrinci,
please verify the BoughtId/SoldId mapping from `*_flashloan_swap` next time.

Thank you.